rules/windows/lateral_movement_evasion_rdp_shadowing.toml

```
@@ -0,0 +1,50 @@
+[metadata]
+creation_date = "2021/04/12"
+maturity = "production"
+updated_date = "2021/04/12"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the modification of the Remote Desktop Protocol (RDP) Shadow registry or the execution of processes
+indicative of active RDP shadowing session. An Adversary may abuse RDP Shadowing feature to spy on other users active
+RDP sessions or even control them.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*", "winlogbeat-*", "logs-windows.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Potential Remote Desktop Shadowing Activity"
+references = [
+"https://bitsadm.in/blog/spying-on-users-using-rdp-shadowing",
+"https://swarm.ptsecurity.com/remote-desktop-services-shadowing/"]
+risk_score = 73
+rule_id = "c57f8579-e2a5-4804-847f-f2732edc5156"
+severity = "high"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Lateral Movement"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+/* Identifies the modification of RDP Shadow registry or
+  the execution of processes indicative of active shadow RDP session */
+any where event.category : ("registry", "process") and
+  registry.path : "HKLM\\Software\\Policies\\Microsoft\\Windows NT\\Terminal Services\\Shadow" or
+  (process.name : ("RdpSaUacHelper.exe", "RdpSaProxy.exe") and process.parent.name : "svchost.exe") or
+  (process.pe.original_file_name : "mstsc.exe" and process.args : "/shadow:*")
```

I don't think we need `registry where process.name : ("RdpSaUacHelper.exe", "RdpSaProxy.exe")` right? In which case, this logic is a little tighter and easier to read

```suggestion
/* Identifies the modification of RDP Shadow registry or
  the execution of processes indicative of active shadow RDP session */

any where 
  (event.category == "registry" and
     registry.path : "HKLM\\Software\\Policies\\Microsoft\\Windows NT\\Terminal Services\\Shadow"
  ) or
  (event.category == "process" and 
     (process.name : ("RdpSaUacHelper.exe", "RdpSaProxy.exe") and process.parent.name : "svchost.exe") or
     (process.pe.original_file_name : "mstsc.exe" and process.args : "/shadow:*")
  )
```

---

it's not needed cuz the execution of "RdpSaUacHelper.exe" or "RdpSaProxy.exe catches the run-time state (registry is supposed to be set before/existing shadow rdp) and in itself is enough to alert. registry change is creation-state detection.

---

👍 just making sure